Source/Workers/decodeDraco.js

```diff
@@ -0,0 +1,119 @@
+define([
+        '../Core/ComponentDatatype',
+        '../Core/defined',
+        '../Core/IndexDatatype',
+        '../Core/RuntimeError',
+        '../ThirdParty/draco-decoder-gltf',
+        './createTaskProcessorWorker'
+    ], function(
+        ComponentDatatype,
+        defined,
+        IndexDatatype,
+        RuntimeError,
+        draco,
+        createTaskProcessorWorker) {
+    'use strict';
+
+    var dracoDecoder;
+
+    function decodeIndexArray(dracoGeometry) {
+        var numPoints = dracoGeometry.num_points();
+        var numFaces = dracoGeometry.num_faces();
+
+        var faceIndices = new draco.DracoInt32Array();
+        var indexArray = IndexDatatype.createTypedArray(numPoints, numFaces * 3);
+
+        for (var i = 0; i < numFaces; ++i) {
+            dracoDecoder.GetFaceFromMesh(dracoGeometry, i, faceIndices);
+
+            var offset = i * 3;
```

Comment:
Old school optimization that won't matter, but why not `offset += 3;` after the assignments to `indexArray` to avoid the multiplication?